### PR TITLE
Refactor: better folder structure and build images with specific configuration

### PR DESCRIPTION
**Pull request by PymZoR**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
 .DS_Store
-data/
-admin/
-log/
+volumes/
+images/buckless-server/buckless-server.config.json
```
Comment (gjuchault) [inline .gitignore]: Missing `images/buckless-server/buckless-server.client.json`

File: images/buckless-manager/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM buckless/manager
+
+COPY ./buckless-manager.manifest.json /usr/src/buckless-manager/manifest.json
```
Comment (gjuchault) [inline images/buckless-manager/Dockerfile]: Missing all images files (`images/iconXX.png`)

**Comment by PymZoR** [comment]
TODO: 
- Move all ngnix config to server.conf
- Create server.conf.example

**Comment by PymZoR** [comment]
@gjuchault APPROVE :goat:

**Pull request merged**